### PR TITLE
Add `oj` Gem to use for JSON parsing

### PR DESCRIPTION
**Pull request by r2bit**
Also added `.ruby-version` (v2.6) and removed Kafka support.

### CODE REVIEW

**Comment by take-five** [comment]
How will you configure Fluentd to use Oj?

**Comment by r2bit** [comment]
> How will you configure Fluentd to use Oj?

In Fluentd config: https://docs.fluentd.org/parser/json#json_parser

**Comment by r2bit** [comment]
!build

**Comment by sm-deployer** [comment]
Built and published `1.14.4-f61ae7b`

**Review by take-five — APPROVED** [review]
👍

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.6
```
Comment (take-five) [inline .ruby-version:1]: Is it the actual Ruby version used by Fluentd?

Comment (r2bit) [reply .ruby-version:1]: Yes. Sadly they are still on 2.6.

**Pull request merged**